pkg/goDB/db_writer.go

```diff
@@ -11,85 +11,43 @@
 package goDB
 
 import (
-	"encoding/binary"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strconv"
-	"time"
 
 	"github.com/els0r/goProbe/pkg/goDB/encoder/bitpack"
 	"github.com/els0r/goProbe/pkg/goDB/encoder/encoders"
 	"github.com/els0r/goProbe/pkg/goDB/storage/gpfile"
+	"github.com/els0r/goProbe/pkg/types"
 	"github.com/els0r/goProbe/pkg/types/hashmap"
 )
 
 const (
 	// QueryLogFile is the name of the query log written by the query package
 	QueryLogFile = "query.log"
-	// MetadataFileName specifies the location of the daily column metadata file
-	MetadataFileName = "meta.json"
 )
 
-// DayTimestamp returns timestamp rounded down to the nearest day
-func DayTimestamp(timestamp int64) int64 {
-	return (timestamp / EpochDay) * EpochDay
-}
-
 // DBWriter writes goProbe flows to goDB database files
 type DBWriter struct {
 	dbpath string
 	iface  string
 
 	dayTimestamp int64
 	encoderType  encoders.Type
-
-	metadata *Metadata
 }
 
 // NewDBWriter initializes a new DBWriter
 func NewDBWriter(dbpath string, iface string, encoderType encoders.Type) (w *DBWriter) {
-	return &DBWriter{dbpath, iface, 0, encoderType, new(Metadata)}
+	return &DBWriter{dbpath, iface, 0, encoderType}
 }
 
 func (w *DBWriter) dailyDir(timestamp int64) (path string) {
-	dailyDir := strconv.FormatInt(DayTimestamp(timestamp), 10)
+	dailyDir := strconv.FormatInt(gpfile.DirTimestamp(timestamp), 10)
```

Comment:
What a random choice by us back then. Instead of just using`fmt.Sprintf`.
```suggestion
	dailyDir := fmt.Sprintf("%d", gpfile.DirTimestamp(timestamp))
```
Or even better, just putting this on line 47. Or doing the whole thing together on line 48.